sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md

```diff
@@ -93,6 +93,19 @@ _The following was run on Windows 10 and Ubuntu Desktop 20.04 environments, with
 
     NOTE: If you are using WSL, do not run these commands from the Windows system drive (e.g. `/mnt/c/`).
 
+    ### Certificates - Important to know
+
+    For sample purposes, Embedded C SDK samples leverage certificates obtained when executing either `New-ArduinoZipLibrary.ps1` or `generate_arduino_zip_library.sh`. 
+    When creating solutions based on the Embedded C SDK you should consider :
+
+    1. For production environments, the IoT Hub certificate must be validated during the TLS handshake.
```

Comment:
```suggestion
    1. The IoT Hub certificate must be validated during the TLS handshake for production environments.
```

Reply:
Again, notice that now we have server cert validation on all our samples, even for prototyping. 
I don't think  we should create space for customers to work with our code in unsafe situation, if they do that they must make that choice themselves. LST, I would suggest removing "For production environment, ".